src/applications/personalization/profile/components/ProfileSubNav.jsx

```diff
@@ -12,7 +12,7 @@ const ProfileSubNav = ({ isInMVI, isLOA3, routes }) => {
   }, []);
 
   return (
-    <nav className="va-subnav" aria-label="Secondary">
+    <nav className="va-subnav" aria-labelledby="subnav-header">
```

Comment:
could you update the comment above the `useEffect` hook?

Reply:
completed